data-sources/wikipedia-wikidata/import_wikidata.sh

```diff
@@ -0,0 +1,88 @@
+    wget "https://query.wikidata.org/bigdata/namespace/wdq/sparql?format=json&query=SELECT ?item?lat?lon WHERE{?item wdt:P31*/wdt:P279*wd:$F;wdt:P625?pt.?item p:P625?loc.?loc psv:P625?cnode.?cnode wikibase:geoLatitude?lat.?cnode wikibase:geoLongitude?lon.}" -O $F.json
+    jq -r '.results | .[] | .[] | [.item.value,.lat.value, .lon.value] | @csv ' $F.json >> $F.txt
+    awk -v qid=$F '{print $0 ","qid}' $F.txt | sed -e 's!"http://www.wikidata.org/entity/!!' | sed 's/"//g' >> $F.csv
+    cat $F.csv >> wikidata_place_dump.csv
+    rm $F.json $F.txt $F.csv
+done < wikidata_place_types.txt
+
+
+# import wikidata places
+
+echo "CREATE TABLE wikidata_place_dump (item text,  lat numeric(11,8), lon numeric(11,8), instance_of text, ont_level integer);"  | psqlcmd
+echo "COPY wikidata_place_dump (item, lat, lon, instance_of) FROM  '/srv/nominatim/Nominatim/data-sources/wikipedia-wikidata/wikidata_place_dump.csv' DELIMITER ',' CSV;"  | psqlcmd
+echo "CREATE TABLE wikidata_places AS SELECT DISTINCT item, instance_of, lon, lat, ont_level FROM wikidata_place_types;" | psqlcmd
+echo "CREATE TABLE wikidata_place_type_levels (place_type text, level integer);" | psqlcmd
+echo "COPY wikidata_place_type_levels(place_type, level) FROM '/srv/nominatim/Nominatim/data-sources/wikipedia-wikidata/wikidata_place_type_levels.csv' DELIMITER ',' CSV HEADER;" | psqlcmd
+echo "UPDATE wikidata_places SET ont_level = level FROM wikidata_place_type_levels WHERE instance_of = place_type;" | psqlcmd
+
+
+# create wikidata language pages with wikipedia links
+
+for i in "${language[@]}"
+do
+   echo "create table geo_earth_wikidata_${i}_pages as select geo_earth_wikidata_places.gt_page_id, geo_earth_wikidata_places.gt_lat, geo_earth_wikidata_places.gt_lon, geo_earth_wikidata_places.page_title, geo_earth_wikidata_places.instance_of, wb_items_per_site.ips_site_page FROM geo_earth_wikidata_places LEFT JOIN wb_items_per_site ON (CAST (( LTRIM(geo_earth_wikidata_places.page_title, 'Q')) AS INTEGER) = wb_items_per_site.ips_item_id) WHERE ips_site_id = '${i}wiki' order by geo_earth_wikidata_places.gt_page_id;" | psqlcmd
+   echo "insert into geo_earth_wikidata_pages select gt_page_id, gt_lat, gt_lon, '${i}', page_title, instance_of, ips_site_page from geo_earth_wikidata_${i}_pages;" | psqlcmd
+done
+
+echo "ALTER TABLE geo_earth_wikidata_pages ADD COLUMN wp_page_title text;" | psqlcmd
+echo "UPDATE geo_earth_wikidata_pages SET wp_page_title = REPLACE(ips_site_page, ' ', '_');" | psqlcmd
+
+
+# add wikidata to wikipedia_article table
+
+echo "UPDATE wikipedia_article SET lat = geo_earth_wikidata_pages.gt_lat, lon = geo_earth_wikidata_pages.gt_lon, wd_page_title = geo_earth_wikidata_pages.page_title, instance_of = geo_earth_wikidata_pages.instance_of FROM geo_earth_wikidata_pages WHERE wikipedia_article_2019.language = geo_earth_wikidata_pages.language AND wikipedia_article_2019.title  = geo_earth_wikidata_pages.wp_page_title AND geo_earth_wikidata_pages.gt_lat IS NOT NULL;" | psqlcmd
```

Comment:
The 2019 in the table name is surprising. What creates the table in the `wikiprocessingdb` database?